types/twine-sugarcube/macro.d.ts

```diff
@@ -69,7 +70,13 @@ export interface MacroContext {
      * The (execution) context object of the macro's parent, or null if the macro has no parent.
      * @since 2.0.0
      */
-    parent: object;
+    parent: null | object;
+
+    /**
+     * The parser instanced that generated the macro call.
+     * @since 2.0.0
+     */
+    parser: object;
```

Comment:
Perhaps it should be declared as `unknown`?

Reply:
`parser` should always contain a reference to the parser instanced that generated the macro call.  Unsure why `unknown` would be of interest.

Reply:
We know it is an object, so it seems incorrect to lose that information.